PubnativeLite/PubnativeLite/Location/PNLiteLocationManager.m

```diff
@@ -48,6 +48,18 @@ - (id)init {
     return self;
 }
 
+static BOOL locationUpdatesEnabled;
++ (BOOL) locationUpdatesEnabled {
+    @synchronized(self) { return locationUpdatesEnabled; }
+}
+
++ (void) setLocationUpdatesEnabled:(BOOL)enabled {
+    @synchronized(self) { locationUpdatesEnabled = enabled; }
+    if (locationUpdatesEnabled) {
+        [[[PNLiteLocationManager sharedInstance]manager]startUpdatingLocation];
```

Comment:
@faresbh Do we need to add an extra check for if the locationservices enabled by the system before updating the location? 🤔

Reply:
No, because im using the manager instance, which is already initialized and does care of that. This will not do anything if location services are not enabled.

Reply:
Coolio 👍